code/src/App.js

```diff
@@ -1,9 +1,90 @@
-import React from 'react'
+/*Outer Dependencies*/
+import React, { useState, useEffect } from "react";
 
+/*Local Dependency*/
+import Form from "./components/Form";
+import List from "./components/List";
+
+import { URL } from "./reusable/urls";
+import { URL_HEARTS } from "./reusable/urls";
+
+/* Main App*/
 export const App = () => {
+  /*States*/
+  const [messageList, setMessageList] = useState([]);
+  const [newMessage, setNewMessage] = useState("");
+
+  /*UseEffect*/
+  useEffect(() => {
+    fetchMessageList();
+    console.log("hi!");
```

Comment:
you can delete the console.log in your code:)

Reply:
You must do it, always 👮🏻

Reply:
Done!